pkg/helpers/helpers.go

```diff
@@ -62,8 +64,9 @@ import (
 const maxConcurrentReconcilesEnvVarName = "MAX_CONCURRENT_RECONCILES"
 
 const (
-	nodeSelectorAnnotation = "open-cluster-management/nodeSelector"
-	tolerationsAnnotation  = "open-cluster-management/tolerations"
+	nodeSelectorAnnotation      = "open-cluster-management/nodeSelector"
+	tolerationsAnnotation       = "open-cluster-management/tolerations"
+	kubeVersionClusterClaimName = "kubeversion.open-cluster-management.io"
```

Comment:
you can actually get kube verison directly from status of managed cluster, right?

Reply:
Yes, I've updated the code as suggested.